# Goal
string io_type # "tool" or "controller"
int16 ionum
int16 value # 1 for ON, 0 for OFF

---
# Result
bool success
string message

---
# Feedback
string status
